api/api.yaml

```diff
@@ -0,0 +1,42 @@
+    openapi: 3.0.0
+    info:
+      title: TranslateApp API
+      description: Api in Golang.
+      version: 0.0.1
+    servers:
+      - url: http://localhost:8080/
+        description: Optional server description, e.g. Main (production) server
+    paths:
+      /languages:
+        get:
+          summary: Returns a list of languages.
+          description: You can take one language from the list as option in translating word.
+          responses:
+            '200':    # status code
+              description: A JSON array of languages
+              content:
+                application/json:
+                  schema: 
+                    type: array
+                    items: 
+                      type: string
+
+      /translate:
+        post:
+          summary: Creates a translation.
+          requestBody:
+            required: true
+              content:
+                application/json:
+                  schema:
+                    type: object
+                      properties:
+                        source:
+                         type: string
+                        target:
+                         type: string
+                        word:
+                         type: string
+        responses: 
```

Comment:
Should be 200 because we don't create anything. Also add 500 here.
Please also take a look at guides from Microsoft and Google how to deal with errors and how to return information on errors.